docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -1,68 +1,189 @@
+The codebase MUST contain documentation of its objectives – like a mission and goal statement – that is understandable by designers and developers so that they can use or contribute to the codebase. |  | See signalen.org and [technical goals](https://github.com/Amsterdam/signals/blob/master/docs/topics/application-design.md).
+The codebase SHOULD contain documentation on its objectives understandable by policy makers and management. | |
+The codebase MAY contain documentation of its objectives for the general public. | |
+
+
+## [Document your code](https://standard.publiccode.net/criteria/documenting.html)
+
+- [ ]
+
+Requirement | meets | links and notes
+-----|-----|-----
+All of the functionality of your codebase – policy as well as source – MUST be described in language clearly understandable for those that understand the purpose of the code. | | The documentation is pretty scattered.
+The documentation of your codebase MUST contain: a description of how to install and run the source code, examples demonstrating the key functionality. | | There are no docs for system administrators. There is documentation on [helm-charts](https://github.com/Signalen/helm-charts), and developer documentation for [frontend](https://github.com/Signalen/frontend/blob/develop/README.md) and [backend](https://github.com/Signalen/backend/blob/master/README.md).
+The documentation of your codebase SHOULD contain: a high level description that is clearly understandable for a wide audience of stakeholders, like the general public and journalists, a section describing how to install and run a standalone version of the source code, including, if necessary, a test dataset, examples for all functionality. |  yes | Signalen.org website.
+There SHOULD be continuous integration tests for the quality of your documentation. | no | Link-checker does not check external links. Markdown in code repositories is not automatically checked.
+The documentation of your codebase MAY contain examples that make users want to immediately start using your codebase. | |
+You MAY use the examples in your documentation to test your code. | |
 
 ## [Use plain English](https://standard.publiccode.net/criteria/understandable-english-first.html)
 
-* Translate the Dutch-only documentation and text in code, so it is also available in English.
-* Ensure dutch names, messages, words in code are localizable.
+- [ ]
+
+Requirement | meets | links and notes
+-----|-----|-----
+All code and documentation MUST be in English. | no | Some workflow contains Dutch states without English translation; If signalen.org is the source for some documentation (and not only a marketing asset), then it should be also available in English.
+Any translation MUST be up to date with the English version and vice-versa. | | See above
+There SHOULD be no acronyms, abbreviations, puns or legal/domain specific terms in the codebase without an explanation preceding it or a link to an explanation. | | Glossary needs expanding.
```

Comment:
I would like to add that trying to avoid _"acronyms, abbreviations, puns or legal/domain specific terms"_ can lead to more misunderstandings between developers and users --- and thus lower quality software less fit for purpose. As remarked above, the glossary needs work.

Reply:
I agree that avoiding all specific terms can lead to confusion. Not sure this is the right place to discuss the standard itself. Maybe open a discussion on https://github.com/publiccodenet/standard/issues for this one?